components/crud-web-apps/jupyter/frontend/src/app/pages/form/form-default/form-default.component.ts

```diff
@@ -106,7 +106,49 @@ export class FormDefaultComponent implements OnInit, OnDestroy {
 
     // Use the custom image instead
     if (notebook.customImageCheck) {
+      // Remove unnecessary images from payloaf
+      delete notebook.imageVSCode;
+      delete notebook.imageRStudio;
       notebook.image = notebook.customImage;
+      if (notebook.serverType == 'vs-code') {
+        // Remove unnecessary images from payloaf
+        delete notebook.imageVSCode;
+        delete notebook.imageRStudio;
+        // Set base URI to / for Istio rewrite
+        notebook.httpRewriteURI = '/';
+      } else if (notebook.serverType == 'rstudio') {
+        // Remove unnecessary images from payloaf
+        delete notebook.imageVSCode;
+        delete notebook.imageRStudio;
+        // Set base URI to / for Istio rewrite
+        notebook.httpRewriteURI = '/';
```

Comment:
Lets also remove the `notebook.httpHeadersRequestSet` and `notebook.httpRewriteURI` from this part of the code as well